doc-content/enterprise-only/installation/clustering-smart-router-install-proc.adoc

```diff
@@ -1,12 +1,10 @@
 [id='clustering-smart-router-install-proc_{context}']
-= Clustering {KIE_SERVERS} with Smart Router
+= Load-balancing {KIE_SERVER} instances with Smart Router
 
-You can use Smart Router to aggregate multiple independent {KIE_SERVER} instances as though they are a single server. It performs the role of an intelligent load balancer because it can route requests to individual {KIE_SERVER}s and aggregate data from different {KIE_SERVER}s. Smart Router uses aliases to perform as a proxy.
-
-This section describes how to install Smart Router and configure it for a Red Hat {PRODUCT_SHORT} runtime environment.
+You can use Smart Router to aggregate multiple independent {KIE_SERVER} instances as though they are a single server. It performs the role of an intelligent load balancer because it can route requests to individual {KIE_SERVER} insances and aggregate data from different {KIE_SERVER} instances. Smart Router uses aliases to perform as a proxy.
```

Comment:
```suggestion
You can use Smart Router to aggregate multiple independent {KIE_SERVER} instances as though they are a single server. It performs the role of an intelligent load balancer because it can route requests to individual {KIE_SERVER} instances and aggregate data from different {KIE_SERVER} instances. Smart Router uses aliases to perform as a proxy.
```